latex/ledger-spec.tex

```diff
@@ -328,6 +353,8 @@ \section{UTxO}
 \begin{itemize}
 \item Each input spent in the transaction must be in the set of unspent
   outputs.
+\item The fee paid by the transaction has to be less than or equal to the
```

Comment:
no, it must be at least the minimum fee.
Maybe also change " ... paid by the transaction ...", it is paid from the UTxO of the sender / creator of the transaction. Maybe just put "The transaction fee is ... "

Reply:
Typo! should say "greater than"